lib/fathead/electron_api/output.txt

```diff
@@ -0,0 +1,590 @@
+app.setLoginItemSettings macOS Windows	A										<section class="prog__container"><ul>\n<li><code class="highlighter-rouge">settings</code> Object\n    <ul>\n<li><code class="highlighter-rouge">openAtLogin</code> Boolean (optional) - <code class="highlighter-rouge">true</code> to open the app at login, <code class="highlighter-rouge">false</code> to remove the app as a login item. Defaults to <code class="highlighter-rouge">false</code>.</li>\n<li><code class="highlighter-rouge">openAsHidden</code> Boolean (optional) - <code class="highlighter-rouge">true</code> to open the app as hidden. Defaults to <code class="highlighter-rouge">false</code>. The user can edit this setting from the System Preferences so <code class="highlighter-rouge">app.getLoginItemStatus().wasOpenedAsHidden</code> should be checked when the app is opened to know the current value. This setting is only supported on macOS.</li>\n</ul>\n</li>\n</ul><ul>\n<li><code class="highlighter-rouge">openAtLogin</code> Boolean (optional) - <code class="highlighter-rouge">true</code> to open the app at login, <code class="highlighter-rouge">false</code> to remove the app as a login item. Defaults to <code
```

Comment:
@rahulxxarora it looks like all the articles link to the same URL, http://electron.atom.io/docs/api/ -- the URLs for article should link to the page where the content came from. i.e. http://electron.atom.io/docs/api/system-preferences/#systempreferencesgetcolorcolor-windows in this case 👍